docs/book/codegen.md

```diff
@@ -5,14 +5,16 @@ generators to create optimized code for production. These generators will
 inspect the provided classes, resolve their dependencies, and generate factories
 based on these results.
 
-> ### Requirements
+> ### Requirements (superseded)
```

Comment:
I think the entire block is an important notice, but I would replace the header "Requirements", because the is no longer a requirement.

Maybe: "zend-code no longer needed"

Reply:
At the moment we have no documentation for different version, therefore I would keep this notice.

@tux-rampage 
Is an extension required for the [migration guide](https://docs.zendframework.com/zend-di/migration/)?

Reply:
I don't see a need for an extension of the migration guide, but I'll double-check